package.json

```diff
@@ -54,7 +55,7 @@
   "scripts": {
     "- client": "# ------------------------",
     "start:client": "vue-cli-service serve",
-    "build": "vue-cli-service build;cp ./client/public/favicon.ico ./assets",
+    "build": "VUE_APP_BUILD_OUTPUT=assets vue-cli-service build && cp ./client/public/favicon.ico ./assets",
```

Comment:
This fixes that duplicate key issue I noticed, removed the dupe key and just set the proper env var here instead.